nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+
+app.get("/numbers/add", (req, res) => {
+  console.log(req.query);
+  let firstNumber,secondNumber;
+
+  if("firstNumber" in req.query){
+    firstNumber=Number(req.query.firstNumber);
+  }
+  if("secondNumber" in req.query){
+    secondNumber=Number(req.query.secondNumber);
+  }
+  const sum =firstNumber+secondNumber
+  res.send(`Sum of ${firstNumber} + ${secondNumber} : ${sum}`);
+});
+
+app.get("/numbers/multiply",(req,res)=>{
+  
+  const firstNumber=Number(req.query.firstNumber);
+  const secondNumber=Number(req.query.secondNumber);
+  const multiply =firstNumber*secondNumber
+  res.send(`Multiplication of ${firstNumber} * ${secondNumber} : ${multiply}`);
+
```

Comment:
Good job. What if I give only first number but not second number or vice-versa? 
Can you handle that and send a nice message to the user?